akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala

```diff
@@ -666,7 +666,7 @@ class NewConnectionPoolSpec extends AkkaSpecWithMaterializer("""
       val sink = if (autoAccept) Sink.foreach[Http.IncomingConnection](handleConnection) else Sink.fromSubscriber(incomingConnections)
 
       val binding =
-        Tcp()
+        Tcp.apply
```

Comment:
To keep consistent with other usages:

```suggestion
        Tcp(system)
```

(Can be reverted to `Tcp()` when akka is fixed in that regard.